relations/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "relations"
+description = "Manage collection of objects"
```

Comment:
Probably need another description than the package `collection`.

Reply:
yes it was a copy of collection's cargo.toml